index.js

```diff
@@ -32,9 +28,18 @@ app.get('/todos', (request, response) => {
   });
 });
 
+app.get('/todos/:id',(request, response)=> {
+  connection.query('SELECT * FROM todos WHERE todoId = ?', [request.params.id], (error, data) => {
+    if (data[0] === undefined) {
+      return response.status(404).json({status: 404, data: error});
```

Comment:
Thank you @hasanshahoud @aboRoma @daanaerts for your feedback. I've made a small change. I hope it's OK now.